src/main.js

```diff
@@ -189,6 +189,11 @@ app.on('ready', function() {
         title: arg.title,
         content: arg.options.body
       });
+
+      /* Todo: add idle here */
```

Comment:
Yes, `notified` message is sent on Windows <= 8.1 only. So this doesn't work with Windows 10.